source/usage-examples/watch.txt

```diff
@@ -0,0 +1,139 @@
+=================
+Watch for Changes
+=================
+
+.. default-domain:: mongodb
+
+Open a Change Stream
+--------------------
+
+You can keep track of changes to data in MongoDB, such as changes to a
+collection, database, or deployment, by opening a **change stream**. A change
+stream allows applications to watch for changes to data and react to them.
+You can open a change stream by calling the ``watch()`` method on
+a ``MongoCollection``, ``MongoDatabase``, or ``MongoClient`` object. The
+change stream returns **change event** documents when they occur.
+
+The ``watch()`` method optionally takes an **aggregation pipeline**  which
+consists of an array of **stages** as the first parameter to filter and
+transform the change event output as follows:. 
+
+.. code-block:: java
+
+   List<Bson> pipeline = Arrays.asList(
+                           Aggregates.match(
+                              Filters.lt("fullDocument.runtime", 15)));
+   ChangeStreamIterable<Document> changeStream = database.watch(pipeline);
+
+To configure options for the change stream, use member methods of the
+``ChangeStreamIterable`` object returned by ``watch()``. Refer to the
+links at the end of this section for more information on the settings
+you can configure.
+
+The ``watch()`` method returns an instance of a ``ChangeStreamIterable``.
```

Comment:
Not mandatory, but I wonder if it might be helpful to describe the class in more detail. For example, here's what I did for MongoIterable, guided by advice from Jeff:
https://github.com/mongodb/docs-java/blob/master/source/usage-examples/find.txt#L21